Sources/UIViewExtensions.swift

```diff
@@ -31,8 +31,8 @@ extension UIView {
 
     /// EZSwiftExtensions, add multiple subviews
     public func addSubviews(_ views: [UIView]) {
-        views.forEach { eachView in
-            self.addSubview(eachView)
+        views.forEach { [weak self] eachView in
```

Comment:
Good catch. I presume this implies the subview has a reference to the parent view and vice versa and due to ARC (Automated ref counting) there is no way to resolve the ref cycle without a weak ref.